### DISCUSSION TITLE
3D Scene - Kinematic moving around joints

### DISCUSSION THREAD

**Discussion by TimUsingAi** [zauberzeug/nicegui#3015] [Ideas / Feature Requests]
### Question

Hi,
is it possible to link or group multiple joints like a arm? For example if I move the elbow that the origin of wrist follows the elbow?
![3d_animated](https://github.com/zauberzeug/nicegui/assets/124078627/280021ed-805c-4e25-8a3a-9574fe874aa0)

**Comment by falkoschindler** [comment]
Hi @TimUsingAi,

That would be an impressive feature, but is currently not implemented. I'll convert this question into a feature request to discuss whether this is feasible. NiceGUI is a UI framework after all, not a physics engine. But maybe there is some lightweight approach or a third-party library we could use.
